appeals/e2e/cypress/e2e/back-office-appeals/linkAppeals.spec.js

```diff
@@ -30,4 +29,22 @@ describe('link appeals', () => {
 			});
 		});
 	});
+
+	it.only('click on the first linked appeal', () => {
```

Comment:
Did you want to leave the `it.only` in? This will means it's the only test that will run within this spec file